### DISCUSSION TITLE
posts/2024/11/04/my-first-post/

### DISCUSSION THREAD

**Discussion by giscus** [GelbEinhalb/gelbeinhalb#2] [General]
# posts/2024/11/04/my-first-post/

This is my first post :)

posts/2024/11/04/my-first-post/

**Comment by GelbEinhalb** [comment]
I hope the comments work :0

**Comment by DrakeChildrenToucher** [comment]
Wie Franklin D. Roosevelt einst sagte: Es gibt immer ein erstes Mal um über den Atlantik deportiert zu werden.

**Reply by GelbEinhalb** [reply]
xD 😭
